README.md

```diff
@@ -21,6 +21,7 @@ postgres:app-links <app>                           # list all postgres service l
 postgres:backup <service> <bucket-name> [--use-iam] # creates a backup of the postgres service to an existing s3 bucket
 postgres:backup-auth <service> <aws-access-key-id> <aws-secret-access-key> <aws-default-region> <aws-signature-version> <endpoint-url> # sets up authentication for backups on the postgres service
 postgres:backup-deauth <service>                   # removes backup authentication for the postgres service
+postgres:backup-s3class <service> <s> <-storage-class> # sets the S3 storage class to be used for backups on the postgres service
```

Comment:
Can you manually regenerate the readme via `make README.md`?

Reply:
That's the way it generates, I can fix the output but idk if there's an issue with the parsing or my subcommand.